website/project/model.py

```diff
@@ -93,8 +90,10 @@ def validate_user(value):
     OSFUser = apps.get_model('osf.OSFUser')
     if value != {}:
         user_id = value.iterkeys().next()
-        if OSFUser.find(Q('_id', 'eq', user_id)).count() != 1:
-            raise ValidationValueError('User does not exist.')
+        try:
+            OSFUser.objects.get(_id=user_id)
```

Comment:
Does this work? Wouldn't this need to be `guids___id=user_id`? Also, you can use exists() to make the query slightly more performant:

```python
if not OSFUser.objects.filter(guids___id=user_id).exists():
    raise ValidatoinError(...)
```

Reply:
Lol, nope.

```
FieldError: Cannot resolve keyword '_id' into field.
```